wrapper/src/test/java/integration/refactored/host/TestEnvironmentProvider.java

```diff
@@ -72,6 +72,7 @@ public Stream<TestTemplateInvocationContext> provideTestTemplateInvocationContex
     final boolean noHikari = Boolean.parseBoolean(System.getProperty("test-no-hikari", "false"));
     final boolean noGraalVm = Boolean.parseBoolean(System.getProperty("test-no-graalvm", "false"));
     final boolean noOpenJdk = Boolean.parseBoolean(System.getProperty("test-no-openjdk", "false"));
+    final boolean noHibernate = Boolean.parseBoolean(System.getProperty("test-no-hibernate", "false"));
```

Comment:
It seems that this flag replaces usual integration tests with hibernate tests. I'd prefer to see additional test configurations in addition to usual integration tests.